dockerfiles/google/Dockerfile

```diff
@@ -43,10 +43,13 @@ RUN apk --no-cache add \
         apache2-utils \
         tzdata \
         jq \
+        make \
+        g++ \
```

Comment:
We should add this, then install the gem, and then remove this in one step -> that way image stays small. Imho not leaving make and other build bits in the final image is much nicer.

Reply:
@stepanstipl Thanks for the suggestion, I had the same idea!